index.html

```diff
@@ -0,0 +1,484 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Avengers | Creative Design Agency</title>
+    <meta name="description" content="Avengers, a Creative Design Agency based out of San Fransisco California, speicalizing in Web Design, UI Design, Icon Design, Branding & Logotype">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    <input type="checkbox" id="mobileNavToggle" class="mobileNavToggle">
+    <header class="l-nav">
+      <nav class="mainNav">
```

Comment:
Please try to be consistent with your naming conventions. Most of your classes are `dash-case`, but this is `camelCase`.

For what it's worth, though it's largely a stylistic choice, I prefer `camelCase` for `id`s and `dash-case` for classes (and other attributes). The reason for that is that `id`s are typically used to reference JavaScript (and not CSS) due to their specificity. Regardless of how you choose to name your `id`s and `class`es, please be consistent.